frontend/src/components/Util/TaskEditors/TaskEditor/index.module.scss

```diff
@@ -110,7 +110,6 @@ img.TaskEditorIcon:focus,
 
 .TaskEditorTag {
   margin-left: 0.5em;
-  font: 1.3em 'Open Sans', sans-serif;
```

Comment:
I added this as part of the [Group Task redesign](https://www.figma.com/file/MoF3jQ5hQ9dT76iZ57i9VN/Group-Work?node-id=878%3A3099), where the tasks have Open Sans font. The main regression we noticed is that the floating task editor has bigger font for some reason even compared to tasks in focus view.

Reply:
maybe removing the font size attribute will do the trick?

Reply:
wait but we don't have tags in group tasks?

Reply:
[It's referring to this tag field](https://github.com/cornell-dti/samwise/issues/339)